src/oss-download/DownloadTool.cs

```diff
@@ -55,8 +53,16 @@ static async Task Main(string[] args)
                 {
                     try
                     {
+                        bool.TryParse(downloadTool.Options["extract"]?.ToString(), out bool doExtract);
+                        bool.TryParse(downloadTool.Options["download-metadata-only"]?.ToString(), out bool metadataOnly);
+                        string targetDirectory = (string)downloadTool.Options["download-directory"];
+                        // are we caching? 
+                        bool doCaching = !string.IsNullOrEmpty(targetDirectory);
```

Comment:
I might have missed this on the last one.  Looks like another cache where caching is implicit depending on if target directory is set.  What if you want the tool to download something in a download folder thats not "." but you don't want cache?